lib/ActivityStreams.js

```diff
@@ -164,16 +173,69 @@ ActivityStreams.prototype = {
    * Sets up the toolbar button that links to the Timeline page.
    */
   _setupButton() {
-    ActionButton({
+    this._button = ActionButton({
       id: "activity-streams-link",
       label: "Timeline",
       icon: data.url("content/img/list-icon.svg"),
       // TODO: Fix url below to about:timeline when that gets implemented.
-      onClick: () => tabs.open("resource://activity-streams/data/content/activity-streams.html#timeline")
+      onClick: () => tabs.open(`${this.options.pageURL}#/timeline`)
     });
   },
 
   /**
+   * The URLs for the app.
+   */
+  get appURLs() {
```

Comment:
If this is getting called on potentially every new dom window opened, we should cache this.
We should also use a `Set`